core/block_svg.js

```diff
@@ -1640,6 +1640,11 @@ Blockly.BlockSvg.prototype.render = function(opt_bubble) {
     }
   }
   Blockly.utils.dom.stopTextWidthCache();
+
+  if (this.pathObject.cursorSvg_ && this.workspace.getCursor().getDrawer()) {
```

Comment:
When would the cursor not have a drawer?

And you can get the cursor before the if, so that you don't have to get it twice.

Reply:
Headless mode and when we just want to use the cursor to traverse the AST. For example the TabNavigateCursor doesn't have a drawer.